packager/mux/mux.go

```diff
@@ -121,14 +121,29 @@ func New(certCache http.Handler, signer http.Handler, validityMap http.Handler,
 	}
 }
 
-var promTotalRequests = promauto.NewCounterVec(
+// Prometheus counter that observes total requests count.
```

Comment:
Comment should be "promRequestsTotal is a promotheus counter ..."
https://golang.org/doc/effective_go.html#commentary